src/index.tsx

```diff
@@ -126,6 +135,12 @@ class AzureAD extends React.Component<IProps, IState> {
         Logger.error(`Error doing login redirect; errorDescription=${this.redirectLoginInfo.errorDesc}, error=${this.redirectLoginInfo.error}`);
       }
     }
+
+    this.checkIfUserAuthenticated();
+  }
+
+  public componentWillReceiveProps(newProps: IProps) {
```

Comment:
Documentation says not to use componentWillReceiveProps (https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops)

Should this instead be called first in componentDidMount instead?

Reply:
Just noticed it already is being called in componentDidMount (i really dislike githubs diff page...). Why do we also need this here then?

Reply:
oh I didn't know that changed! On second thought I don't think componentWillReceiveProps  (or now componentDidUpdate) is necessary